README.md

```diff
@@ -1,16 +1,280 @@
 and the Eureka Service Discovery from the [Spring Cloud Netflix](https://github.com/spring-cloud/spring-cloud-netflix) technology stack.
 
+This fork also demostrates the use of free distributed tracing with Tanzu Observability by Wavefront, which provides cloud-based monitoring of  Spring Boot applications with 5 days of history.
+
+
+
+## Understanding the Spring Petclinic application
+
+[See the presentation of the Spring Petclinic Framework version](http://fr.slideshare.net/AntoineRey/spring-framework-petclinic-sample-application)
+
+[A blog bost introducing the Spring Petclinic Microsevices](http://javaetmoi.com/2018/10/architecture-microservices-avec-spring-cloud/) (french language)
+
+You can then access petclinic here: http://localhost:8080/
+
+![Spring Petclinic Microservices screenshot](./docs/application-screenshot.png?lastModify=1596391473)
+
+**Architecture diagram of the Spring Petclinic Microservices**
+
+![img](./docs/microservices-architecture-diagram.jpg?lastModify=1596391473)
+
+
+
+## Compiling and pushing to Cloud Foundry:
+
+The samples below are using Tanzu Application Service (previously Pivotal Cloud Foundry) as the target Cloud Foundry deployment, some adjustments may be needed for other Cloud Foundry distributions.
+
+Generate your free wavefront token by running one of the apps, for example:
+
+```bash
+cd spring-petclinic-api-gateway
+mvn spring-boot:run
+```
+
+You will see something like this in the logs:
+
+```
+A Wavefront account has been provisioned successfully and the API token has been saved to disk.
+
+To share this account, make sure the following is added to your configuration:
+
+	management.metrics.export.wavefront.api-token=2e41f7cf-1111-2222-3333-7397a56113ca
+	management.metrics.export.wavefront.uri=https://wavefront.surf
+
+Connect to your Wavefront dashboard using this one-time use link:
+https://wavefront.surf/us/AAA4s5f8xJ9yD
+
+```
+
+You free account has now been created.
+
+Create a user-provided service for Wavefront using the data above. For example:
+
+```
+cf cups -p '{"uri": "https://wavefront.surf", "api-token": "2e41f7cf-1111-2222-3333-7397a56113ca", "application-name": "spring-petclinic-cloudfoundry", "fremium": "true"}' wavefront
+```
+If your operator deployed the wavefront proxy in your Cloud Foundry environment, point the URI to the proxy instead. You can obtain the value of the IP and port by creating a service key of the wavefront proxy and viewing the resulting JSON file. 
+
+Contine with creating the services and deploying the application's microservices. A sample is available at `scripts/deployToCloudFoundry.sh`. Note that some of the services' plans may be different in your environment, so please review before executing. For example, you want want to fork [https://github.com/odedia/spring-petclinic-microservices-config.git]https://github.com/odedia/spring-petclinic-microservices-config.git if you want to make changes to the configuration.
```

Comment:
I would prefer a separate repo to avoid any lockin and regression tests between the two repos. They seem to be going in slightly different directions. It will also allow us to experiment more with the cloud-based config without impacting the existing microservices repo.

Reply:
Ok thus I've create the https://github.com/spring-petclinic/spring-petclinic-cloud-config repo. I let you published your config.